wnfs/examples/private.rs

```diff
@@ -4,7 +4,7 @@
 use chrono::Utc;
 use libipld::Cid;
 use rand::{thread_rng, RngCore};
-use std::rc::Rc;
+use std::{collections::BTreeSet, rc::Rc};
```

Comment:
Unused import

Reply:
Same for other doc examples

Reply:
No clue. Looks unresolved for now. https://github.com/rust-lang/cargo/issues/6424

Reply:
cc @zeeshanlakhani. Any idea?